nc_test4/tst_filter.sh

```diff
@@ -3,6 +3,8 @@
 if test "x$srcdir" = x ; then srcdir=`pwd`; fi
 . ../test_common.sh
 
+set -x
```

Comment:
Probably don't want to leave set -x on, as it will cause a lot of output on make check.